signature-aggregator/main/main.go

```diff
@@ -105,9 +106,22 @@ func main() {
 		logger.Fatal("Failed to create message creator", zap.Error(err))
 		panic(err)
 	}
-	signatureAggregator := aggregator.NewSignatureAggregator(network, logger, messageCreator)
 
-	api.HandleAggregateSignaturesByRawMsgRequest(logger, signatureAggregator)
+	registry := metrics.Initialize(cfg.MetricsPort)
+	metrics_ := metrics.NewSignatureAggregatorMetrics(registry)
```

Comment:
We haven't used trailing/leading underscores in our Go codebases. Let's use a different name here.

Reply:
addressed in https://github.com/ava-labs/awm-relayer/pull/398/commits/4feaa81ed88e00952e100b7f4aec4b44e7fab61e